taxi_demo/README.md

```diff
@@ -0,0 +1,72 @@
+# NYC Taxi Demo
+
+This demo contains an end-to-end data analysis and machine learning pipeline using publicly-available data.
+
+## Data 
+
+Yellow taxi trip data from the [NYC Taxi and Limousine Comission (TLC)](https://www1.nyc.gov/site/tlc/about/tlc-trip-record-data.page). The data is [available on S3](https://registry.opendata.aws/nyc-tlc-trip-records-pds/) at `s3://nyc-tlc/`.
+
+The data dictionary for the yellow trip data is available [here](https://www1.nyc.gov/assets/tlc/downloads/pdf/data_dictionary_trip_records_yellow.pdf). Note that this dictionary is for the latest release of data (2019), and the schema/data has changed a few times over the years.
+
+## Running in Saturn Cloud
+
+### Environment / Images
+
+These examples should work with Saturn's default images, but you can also [build an image yourself](https://www.saturncloud.io/docs/getting-started/setup/customizing-environments/) using the requirement files provided.
+
+The `environment.yml` and `gpu_environment.yml` files contain the the conda environments for CPU and GPU environments. ML tasks that utilize RAPIDS require the GPU environment.
+
+### Setup Jupyter workspace
+
+To start running this demo inside Saturn Cloud, you can [create a new Jupyter workspace](https://www.saturncloud.io/docs/getting-started/spinning/jupyter/). If you created a custom image, make sure to select it in the "Image" field. Under advanced settings, include the following in the "Start Script" to specify which S3 path to write data to.
+
+```
+export TAXI_S3='s3://saturn-titan/nyc-taxi'
+```
```

Comment:
yeah it does, i remember in coffee chat a few days ago @hhuuggoo mentioned that we will probably get rid of the environment variable field because it doesnt always work as expected

Reply:
:grimacing: ok I definitely would like to be involved when we do that, I missed that comment